packages/eslint-plugin/src/rules/no-html-links.ts

```diff
@@ -35,7 +35,6 @@ export default createRule<Options, MessageIds>({
     type: 'problem',
     docs: {
       description: 'enforce using Docusaurus Link component instead of <a> tag',
-      recommended: false,
```

Comment:
These are no longer required in types for rules, and it looks to me that these weren't used anywhere in Docusaurus code.